code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -214,45 +219,46 @@
 
 	var/old_heat_capacity = air.heat_capacity()
 	var/reaction_energy
-	var/mediation = 100*(air.heat_capacity()-(cached_gases[/datum/gas/plasma][MOLES]*cached_gases[/datum/gas/plasma][GAS_META][META_GAS_SPECIFIC_HEAT]))/(air.total_moles()-cached_gases[/datum/gas/plasma][MOLES]) //This is the average heat capacity of the mixture,not including plasma.
+
+	var/mediation = FUSION_MEDIATION_FACTOR*(air.heat_capacity()-(cached_gases[/datum/gas/plasma][MOLES]*cached_gases[/datum/gas/plasma][GAS_META][META_GAS_SPECIFIC_HEAT]))/(air.total_moles()-cached_gases[/datum/gas/plasma][MOLES]) //This is the average heat capacity of the mixture,not including plasma.
 	var/gas_power = 0
 	for (var/id in cached_gases)
 		gas_power += cached_gases[id][GAS_META][META_GAS_FUSION_POWER]*cached_gases[id][MOLES]
-	var/plasma_fused = 0
-	var/power_ratio = min(gas_power/mediation,100)//100 is a lot, we really don't want to go over this.
-	if (power_ratio > 10) //Super-fusion. Fuses everything into one big atom which then turns to tritium instantly. Very dangerous, but super cool.
+
+	var/power_ratio = gas_power/mediation
+
+	if (power_ratio > FUSION_HIGH_TIER) //Super-fusion. Fuses everything into one big atom which then turns to tritium instantly. Very dangerous, but super cool.
 		var/gases_fused = air.total_moles()
-		reaction_energy += gases_fused*PLASMA_BINDING_ENERGY*(gas_power/(mediation*100))
+		reaction_energy += gases_fused*PLASMA_BINDING_ENERGY*(gas_power/(mediation*FUSION_MEDIATION_FACTOR))
```

Comment:
A change I've thought about: Making reaction_energy overall higher for all forms of fusion (this means higher temps), to account for the fact that fusion has way higher gas requirements